src/views/home/components/bridge-form/bridge-form.view.tsx

```diff
@@ -55,19 +56,28 @@ const BridgeForm: FC<BridgeFormProps> = ({ account, formData, resetForm, onSubmi
     status: "pending",
   });
 
+  const getEtherToken = (chain: Chain): Token | undefined => {
+    return getChainTokens(chain).find((token) => token.address === ethers.constants.AddressZero);
+  };
```

Comment:
We could remove this function and use `constants.getEtherToken` directly, but this feels more correct.